testing/src/main/java/org/oppia/android/testing/TestDispatcherModule.kt

```diff
@@ -85,7 +85,6 @@ class TestDispatcherModule {
     return if (isOnRobolectric) robolectricFactoryProvider.get() else espressoFactoryProvider.get()
   }
 
-  @Provides
```

Comment:
Why are we not removing this completely from here?

Reply:
I forgot to remove this completely I'll push this in a commit that with above kdoc you suggest